invenio_rdm_records/resources/serializers/schemaorg/schema.py

```diff
@@ -187,7 +187,7 @@ class SchemaorgSchema(BaseSerializerSchema, CommonFieldsMixin):
     dateModified = fields.Method("get_modification_date")
     temporal = fields.Method("get_dates")
     inLanguage = fields.Method("get_language")
-    contentSize = fields.Method("get_size")
```

Comment:
minor: This is used in the `MediaObject` type: https://schema.org/MediaObject

we can keep it as a duplicate (similar to `creator/author`)